www/includes/easyparliament/templates/html/mp/divisions.php

```diff
@@ -0,0 +1,206 @@
+                                    <input type="submit" class="button small" value="Make it happen!">
+                                </p>
+                            </form>
+                        <?php } ?>
+
+                        <?php if ($has_voting_record) { ?>
+
+                            <?php foreach ($policydivisions as $policy) { ?>
+
+                                <?php if ( isset($policyinformation) ) { ?>
+                                    <div class="panel policy-votes-hero" style="background-image: url('<?php echo $policyinformation[$policy['policy_id']]['image']; ?>');">
+                                        <h2><?php echo $policyinformation[$policy['policy_id']]['title']; ?></h2>
+                                        <p>Someone who voted for this agreed that <?php echo $policyinformation[$policy['policy_id']]['description']; ?>.</p>
+                                        <?php if ( $policyinformation[$policy['policy_id']]['image_source'] ) { ?>
+                                        <span class="policy-votes-hero__image-attribution">
+                                            Photo:
+                                            <a href="<?php echo $policyinformation[$policy['policy_id']]['image_source']; ?>">
+                                                <?php echo $policyinformation[$policy['policy_id']]['image_attribution']; ?>
+                                            </a>
+                                            <a href="<?php echo $policyinformation[$policy['policy_id']]['image_license_url']; ?>">
+                                                <?php echo $policyinformation[$policy['policy_id']]['image_license']; ?>
+                                            </a>
+                                        </span>
+                                        <?php } ?>
+                                    </div>
+                                <?php } ?>
+
+                                <?php if ( isset($positions[$policy['policy_id']]) ) { ?>
+                                    <div class="panel">
+                                        <h3 class="policy-vote-overall-stance">
+                                            <?= $full_name ?> voted <?= $positions[$policy['policy_id']]['voted'] ?> <?= $policies[$policy['policy_id']] ?>
+                                        </h3>
+
+                                        <?php if ( DEVSITE ) { ?>
+                                        <p class="policy-vote-agree-disagree">
+                                            <button class="button">I agree with this MP</button>
+                                            <button class="button button--negative">I disagree with this MP</button>
+                                        </p>
+                                        <?php } ?>
+
+                                        <h3 class="policy-votes-list-header">Key votes about <?= $policies[$policy['policy_id']] ?>:</h3>
```

Comment:
This should be All votes by default (see JS comment elsewhere), change to Key votes if JS is working, then change back to All when someone clicks the see all votes link :)